insights/parsers/system_time.py

```diff
@@ -174,26 +193,45 @@ class NtpTime(Parser):
 
     """
     def parse_content(self, content):
-        funcname = [
-            'ntp_gettime',
-            'ntp_adjtime'
-        ]
         result = {}
-        return_code_re = re.compile(r'returns code (\d+) ')
-        status_code_re = re.compile(r'\((.*)\)')
+        return_code_re = re.compile(r'(?P<func>ntp_(?:get|adj)time)\(\) returns code (?P<code>\d+) ')
+        status_code_re = re.compile(r'status (?P<status>0x[0-9a-f]{4}) \((?P<flags>.*)\)')
+        time_data_re = re.compile(r'time (?P<timecode>[0-9a-f]{8}\.[0-9a-f]{8}) +(?P<timestamp>\w{3}, \w{3} \d+ .*),')
+        value_re = re.compile(r'(?P<keyword>[A-Za-z][A-Za-z ]+) (?P<value>\d+(?:\.\d+|x[0-9a-f]+)? ?(?:us|ppm|s|\(.*\))?)')
```

Comment:
Going with converting them to integer.  However, we're now a bit inconsistent with the other parts of the code, where the adjtime and gettime results and the stats value are returned as strings.  I'm going to leave them as is but this is a little awkward.